usaspending_api/etl/award_helpers.py

```diff
@@ -0,0 +1,80 @@
+from django.db import connection
+
+
+def update_awards():
+    """
+    Update award data based on the information in their child transactions.
+
+    Note: Obviously the raw SQL is not ideal. That said, the complex update
+    of award fields based on the earliest, latest, and aggregate values
+    of the child transactions was problematic to do in a set-based way
+    via the ORM. These updates do need to be set-based, as looping through
+    and updating individual award records would be an ETL bottleneck.
+
+    TODO: modify this to work with specific awards, so we aren't updating
+    every award, all the time
+    """
```

Comment:
Oooooh--good idea. That is way simpler than what I was thinking of. Thanks for the suggestion!